lib/i18n/tasks/data/tree/siblings.rb

```diff
@@ -10,6 +10,9 @@ module I18n::Tasks::Data::Tree
   # in case of an empty parent (nil) it represents a forest
   # siblings' keys are unique
   class Siblings < Nodes # rubocop:disable Metrics/ClassLength
+    # Ref: http://cldr.unicode.org/index/cldr-spec/plural-rules
+    CLDR_CATEGORY_KEYS = %w[zero one two few many other].freeze
```

Comment:
This could maybe reuse a similar definition from I18n::Tasks::PluralKeys

Reply:
Refactored